stories/topics/ref-aap-gcp-automation-performance.adoc

```diff
@@ -8,9 +8,14 @@ Automating outside of the boundaries of these criteria is not supported, but can
 [cols="30%,30%",options="header"]
 |====
 | Metric | Threshold
-| Concurrent Jobs | -
-| Forks per job | -
+| Concurrent Jobs | 10
+| Forks per job | 10
 |====
 
-TBD
-
+[NOTE]
+====
+{AAPonGCP} is driven using three m5.large instances, two of which run {PlatformName} {ControllerName} and one which runs {HubName}. The {ControllerName} instances collectively support a standard workload for 100 managed active nodes. 
+Testing this operating criteria by Red Hat found support for up-to 10 forks each.  
```

Comment:
```suggestion
Red Hat has tested the operating criteria for up-to 10 forks each.  
```
Suggesting that this be re-phrased.  My attempt could be misinterpreting the sentence, though.